test/emoji.js

```diff
@@ -0,0 +1,39 @@
+import assert from 'assert';
```

Comment:
Instead of adding a new `emoji.js` test file, please add these tests to `test/bidi.js`.